libcontainer/rootfs_linux.go

```diff
@@ -152,12 +153,24 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		}
 		return nil
 	case "tmpfs":
+		copyUp := m.Extensions&configs.EXT_COPYUP == configs.EXT_COPYUP
+		tmpDir := ""
 		stat, err := os.Stat(dest)
 		if err != nil {
 			if err := os.MkdirAll(dest, 0755); err != nil {
 				return err
 			}
 		}
+		if copyUp {
+			tmpDir, err = ioutil.TempDir("/run", "runctmpdir")
```

Comment:
Why are you using /run here? Shouldn't this be under the machines configured temp dir and not fillup space on /run?